README.md

```diff
@@ -1,2 +1,158 @@
-# budget
-Budget App with Rails
+<a name="readme-top"></a>
+
+<div align="center">
+  <h3><b>Budget App</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+    - [Live Version](#live-version)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+  - [Install](#install)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Budget App <a name="about-project"></a>
+
+**Budget App** is an web app built with rails and uses postgresql. The user can log in, create new categories and new records for their expenses. Each category has a name and an icon, and each expense has a name, amount and a category. The user also can see the amount for each category and the total amount for all categories.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://ruby.org/">Ruby</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+- **The page is built with Ruby on Rails**
+- **The desing is for mobile first approach**
+- **For handling user's log and register the App uses device gem**
+
+### Live Version <a name="live-version">
+[Budget App](https://budget-aody.onrender.com/)
+</a>
+
```

Comment:
### VIDEO PRESENTATION

- In your video, you've not shown any piece of code that you're proud of from your code editor, please, re-record another video to add this missing part of the project requirement to improve your video.